src/utils/utilities.jl

```diff
@@ -403,3 +400,62 @@ function gaussian_smooth(arr::AbstractArray, sigma::Int = 1)
 
     return smoothed_arr
 end
+
+"""
+    periods_are_commensurate(dt_large::Dates.Period, dt_small::Dates.Period)
```

Comment:
Is commensurate the right word here? Aren't we basically looking to see if one is an integer multiple of the other?

Reply:
You are right!

I started thinking about commensuare (two non-zero real numbers a and b are said to be commensurable if their ratio ⁠a/b⁠ is a rational number), but at the end I am not checking that, I am checking if they are integer multiple of the other